### PR TITLE
Rename max_iob.json to preferences.json

### PR DESCRIPTION
**Pull request by mddub**

### CODE REVIEW

File: docs/reference/oref0/oref0-mint-max-iob.md [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
 
 ## Help
-  Usage:
-oref0-mint-max-iob max_iob <max_iob> [max_iob.json]
+Usage:
+oref0-mint-max-iob.sh <max_iob> [preferences.json]
 
-oref0-mint-max-iob help - this message
-Print a perfect max_iob.json.
+oref0-mint-max-iob.sh help - this message
```
Comment (scottleibrand) [inline docs/reference/oref0/oref0-mint-max-iob.md]: We don't want the .sh here.  When installed, the oref0-mint-max-iob command is run without it:

edison@edison-rl ~ $ which oref0-mint-max-iob
/usr/local/bin/oref0-mint-max-iob
edison@edison-rl ~ $ which oref0-mint-max-iob.sh
edison@edison-rl ~ $

Comment (mddub) [inline docs/reference/oref0/oref0-mint-max-iob.md]: Good point, fixed!

File: docs/reference/oref0/oref0-mint-max-iob.md [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+Print a perfect preferences.json.
 
 
 Examples:
-$ oref0-mint-max-iob 2
+$ oref0-mint-max-iob.sh 2
```
Comment (scottleibrand) [inline docs/reference/oref0/oref0-mint-max-iob.md]: Same here.

File: docs/reference/oref0/oref0-mint-max-iob.md [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
 {
   "max_iob": 2
-}
+  }
 
-$ oref0-mint-max-iob 2 foo.json
-max_iob 2 saved in foo.json
-bewest@hither:~/src/openaps/oref0$ 
+  $ oref0-mint-max-iob.sh 2 foo.json
```
Comment (scottleibrand) [inline docs/reference/oref0/oref0-mint-max-iob.md]: And here.

**Pull request merged**